package_app.sh

```diff
@@ -35,7 +38,7 @@ export RUSTFLAGS="-C link-arg=-Wl,-rpath,\$ORIGIN/lib/native"
 echo "RUSTFLAGS=${RUSTFLAGS}"
 
 # Set LD_LIBRARY_PATH as needed for building and testing.
-export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${RUST_LIB_DIR}:${JAVA_LIB_DIR}"
+export LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-}:${RUST_LIB_DIR}:${JAVA_LIB_DIR}"
```

Comment:
Is it needed to include the value of an already set LD_LIBRARY_PATH? Won't it cause surprising effects?

Reply:
I don't think it can cause any issues. I'm using this to avoid complaining about uninitialized variable.